libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -382,16 +382,16 @@ - (void)sendPendingRequests {
     }
 }
 
-- (void)notifyDelegateOfResponse:(id<SFRestDelegate>)delegate request:(SFRestRequest *)request data:(id)data {
-    if ([delegate respondsToSelector:@selector(request:didLoadResponse:)]) {
-        [delegate request:request didLoadResponse:data];
+- (void)notifyDelegateOfResponse:(id<SFRestDelegate>)delegate request:(SFRestRequest *)request data:(id)data rawResponse:(NSURLResponse *)rawResponse {
+    if ([delegate respondsToSelector:@selector(request:didLoadResponse:rawResponse:)]) {
+        [delegate request:request didLoadResponse:data rawResponse:rawResponse];
```

Comment:
@bhariharan @khawkins Can we get a fallback to the non-raw-response version of this? I.e.
`else if ([delegate respondsToSelector:@selector(request:didLoadResponse:)` to support backwards compatibility?

Reply:
@asikora-sf Yeah, it's done [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/2210).